api/jwt_test.go

```diff
@@ -0,0 +1,34 @@
+package api
+
+import (
+	"testing"
+
+	"go.mongodb.org/mongo-driver/bson/primitive"
+
+	"github.com/NebulousLabs/skynet-accounts/database"
+)
+
+// TestToken ensures that IssueToken() and ValidateToken() function properly.
+// TODO This test is too basic and needs to be expanded.
+func TestToken(t *testing.T) {
+	id, err := primitive.ObjectIDFromHex("5fbd557f0518ba7ae9267e3b")
```

Comment:
We should hard-code an expected ID to match against, seems important to make sure it shouldn't ever change.

Reply:
I'm not sure what you mean. Add a record with a known id to the DB and look for it in the tests? The tests will be run against a brand new, empty DB, so we'd have to insert it first. And this execution path is already tested when we test the `add user` functionality.

Reply:
Thought we were generating IDs, read this way too fast